content/docs/tutorials/interactivity/get-familiar.md

```diff
@@ -0,0 +1,59 @@
+---
+$title: Getting familiar with the starter code
+$order: 1
+toc: true
+---
+
+[TOC]
+
+## AMP boilerplate
+An AMP page is an HTML page with some restrictions for reliable performance. AMP pages have a bit of special markup that identifies it as an AMP page.
+
+A bare bones AMP page looks like this:
+
+```html
+<!doctype html>
+<html amp>
+ <head>
+   <meta charset="utf-8">
+   <link rel="canonical" href="hello-world.html">
+   <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+   <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+   <script async src="https://cdn.ampproject.org/v0.js"></script>
+ </head>
+ <body>Hello World!</body>
+</html>
+```
+
+## AMP components
+
+The tutorial's starter code (`static/index.html`) builds off the bare bones AMP page with its page content (images, text, etc.) as well as including a few AMP components:
```

Comment:
link to https://github.com/googlecodelabs/advanced-interactivity-in-amp/blob/master/static/index.html